### PR TITLE
Animations Dummy Request

### PR DESCRIPTION
**Pull request by AangNaruto**
For Ian's and my eyes only haha

### CODE REVIEW

File: SGoopas/Assets/Scripts/Game/2D/2D Player States/ExitLevel2D.cs [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        }
+
+        public override void MoveRight()
+        {
+            // No-op.
+        }
+
+        public override void Update()
+        {
+            SetState(new ExitLevel2D(this));
```
Comment (AangNaruto) [inline SGoopas/Assets/Scripts/Game/2D/2D Player States/ExitLevel2D.cs]: What? o.o

File: SGoopas/Assets/Scripts/Game/2D/2D Player States/Base2DState.cs [hunk 2]
```diff
@@ -213,6 +213,16 @@ protected bool IsGrounded
                 }
             }
         }
+        public void Freeze()
```
Comment (AangNaruto) [inline SGoopas/Assets/Scripts/Game/2D/2D Player States/Base2DState.cs]: Move to ExitLevel2D state

File: SGoopas/Assets/Scripts/Game/Controller/ControllerConfigure.cs [hunk 3]
```diff
@@ -49,6 +49,7 @@ private void ConfigureControls()
 	// Use this for initialization
 	void Start () {
         controller = new Controller();
+        print("reeee");
```
Comment (AangNaruto) [inline SGoopas/Assets/Scripts/Game/Controller/ControllerConfigure.cs]: Don't forget to remove

File: SGoopas/Assets/Scripts/Game/2D/EndTrigger.cs [hunk 4]
```diff
@@ -4,8 +4,29 @@
...
+    {
+        anim = GetComponent<Animator>();
+        //player = MainObjectContainer.Instance.Player2D;
+    }
+
     void OnTriggerEnter2D(Collider2D other)
     {
+        ExitHandler.TriggerExit();
+        anim.SetBool("exit", true);
+        StartCoroutine(ExitLevel());
```
Comment (AangNaruto) [inline SGoopas/Assets/Scripts/Game/2D/EndTrigger.cs]: This is what's causing the multiple level loading, I think. As the player swirls around and collides with the trigger, this coroutine starts a couple times

**Comment by Encheat** [comment]
Fixed the changes, tweaked the exit time so you get a nicer transition to the next level.

**Pull request merged**